server/config/config.env

```diff
@@ -0,0 +1,2 @@
+PORT=5000
+NODE_ENV=development
```

Comment:
.env should not be uploaded to source control! we put secrets there.
add gitignore and make sure that this file is included

Reply:
i already have .gitignor with .env inside I can see it from the beginning. are you sure about it?